core/src/main/scala/org/apache/spark/api/python/PythonRDD.scala

```diff
@@ -874,10 +890,14 @@ private class PythonAccumulatorParam(@transient private val serverHost: String,
 private[spark] class PythonBroadcast(@transient var path: String) extends Serializable
   with Logging {
 
-  /**
-   * Read data from disks, then copy it to `out`
-   */
-  private def writeObject(out: ObjectOutputStream): Unit = Utils.tryOrIOException {
+  val queue = new ReferenceQueue[File]()
+  val phantomReferences = new ListBuffer[FilePhantomReference]()
+  val threadName = "WeakReference"
+/**
```

Comment:
can you fix the indentation here, and add proper blank lines (line before the doc, not after)